zimports.py

```diff
@@ -182,20 +185,20 @@ def _write_source(
                 ):
                     buf.append("")
                 previous_import = import_node
-                buf.append(_write_import(import_node))
+                buf.extend(_write_import(import_node, max_line_length=max_line_length))
```

Comment:
is there a time that _write_import() would be called *without* max_line_length being passed (and if so, why?)   therefore it should be positional required argument